docker/py-vnc/Dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG BASE_IMAGE=renku/renkulab-py:latest
+FROM ${BASE_IMAGE}
+
+LABEL maintainer="Swiss Data Science Center <info@datascience.ch>"
+
+USER root
+
+RUN apt-get update \
+    && apt-get install -yq --no-install-recommends \
+        dbus-x11 \
+        firefox \
+        net-tools \
+        less \
+        xfce4 \
+        xfce4-panel \
+        xfce4-session \
+        xfce4-settings \
+        xorg \
+        xubuntu-icon-theme \
+        xterm \
+    && apt-get autoremove --purge \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /tmp/* \
+    && find /var/log -type f -exec cp /dev/null \{\} \;
+
+# Get noVNC
+RUN cd /opt && \
+    curl -sSfL https://github.com/novnc/noVNC/archive/v1.1.0.tar.gz | tar -zxf -
+
+
+# Patch novnc to use correct path to websockify (defaults to /)
+# Note if you use vnc.html you will need to patch ui.js to use the correct path
+# and also to override localstorage which may store an incorrect path from a
+# different session. Also resize server instead of scaling client
+RUN sed -i.bak \
```

Comment:
I wonder if having a configuration file that you copy in to the container would be more maintainable? This is pretty hard to read atm

Reply:
Fair point. I think that's what Eric did originally. I can add that / change the set up.